### PR TITLE
New article: how to use git as a database

### PR DESCRIPTION
**Pull request by josecelano**

### CODE REVIEW

File: docs/015_How-to-use-Git-as-a-database.md [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+```s
+cd /tmp
+mkdir test
+cd test
+git init
+echo 'test content' | git hash-object -w --stdin
+d670460b4b4aece5915caf5c68d12f560a9fe3e4
+```
+
+The the last command output is the sha1 of the new object. You can get the object content back using that sha1:
```
Comment (cgbosse) [inline docs/015_How-to-use-Git-as-a-database.md:46]: The "the" last command output is

Duplicated word

File: docs/015_How-to-use-Git-as-a-database.md [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+
+In those cases, you should have to define an object like a "pool of counters" and store all the values in the same object (branch).
+
+If we do not have any invariant between objects you can store each object in a different orphan branch. That way you can reduce conflicts and have a better performance. On the contrary, if you have some invariants with more than one object then you have to store the whole aggregate in the same branch, so you make sure we do not have any transactional inconsistencies. You could decide the storage method depending on the invariants declared in application.
+
+I recommend reading [these articles](https://www.dddcommunity.org/library/vernon_2011/) to know the trade-offs of aggregate design.
+
+### Atomic Git push
+
+Sometimes you will need to update the object but also to create other commits in others branches atomically. For example, if you want to update two object at the same time. You could want to update both of them or neither of them. That is to say, you could want to have transactions.
```
Comment (cgbosse) [inline docs/015_How-to-use-Git-as-a-database.md:383]: ally. For example, if you want to update two object "s" at the same time. 

Missing and s

Comment (josecelano) [reply docs/015_How-to-use-Git-as-a-database.md]: Fixed

File: docs/015_How-to-use-Git-as-a-database.md (hunk 1 above)
Comment (josecelano) [reply docs/015_How-to-use-Git-as-a-database.md]: Fixed.

**Pull request merged**